bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -117,30 +122,99 @@ public void componentShown(final ComponentEvent componentEvent) {
+     */
+    private boolean selectionIsAutoTarget() {
+        return _curSelection.flatMap(either -> either.foldLeft(t -> {
+            final TargetEnvironment env = getDataObject().getTargetEnvironment();
+            return env.getGuideEnvironment().getOptions().exists(gg -> gg.isAutomatic() && gg.containsTarget(t));
+        })).getOrElse(false);
+    }
+
+    /**
+     * Auxiliary method to determine if the current selection is a user target.
+     */
+    private boolean selectionIsUserTarget() {
+        return _curSelection.flatMap(either -> either.foldLeft(t -> {
+            final TargetEnvironment env = getDataObject().getTargetEnvironment();
+            return env.getUserTargets().contains(t);
+        })).getOrElse(false);
+    }
+
+    /**
+     * Auxiliary method to set the selection to the specified target.
+     */
+    private void setSelectionToTarget(final SPTarget t) {
+        _curSelection = new Some<>(Either.left(t));
+    }
+
+    /**
+     * Update the UI components when a target becomes selected.
+     */
+    private void updateUIForTarget() {
+        final boolean editable     = OTOptions.areRootAndCurrentObsIfAnyEditable(getProgram(), getContextObservation());
+        final boolean notBase      = !selectionIsBasePosition();
+        final boolean notAuto      = !(selectionIsAutoGroup() || selectionIsAutoTarget());
+        final boolean isGuideStar  = _curSelection.exists(Either::isLeft) && !(selectionIsBasePosition() || selectionIsUserTarget());
+
+        _w.removeButton.setEnabled (editable && notBase && notAuto);
+        _w.primaryButton.setEnabled(editable && isGuideStar && notAuto);
+        _w.pasteButton.setEnabled(editable && notAuto);
+        _w.duplicateButton.setEnabled(editable && isGuideStar && notAuto);
+        updateDetailEditorEnabledState(editable && notAuto);
+    }
+
+    /**
+     * Update the UI components when a group becomes selected.
+     */
+    private void updateUIForGroup() {
+        final boolean editable = OTOptions.areRootAndCurrentObsIfAnyEditable(getProgram(), getContextObservation());
+        final boolean notAuto   = !selectionIsAutoGroup();
+        _w.removeButton.setEnabled(editable && notAuto);
+        _w.primaryButton.setEnabled(editable);
+        _w.pasteButton.setEnabled(editable && notAuto);
+        _w.duplicateButton.setEnabled(editable);
+    }
+
+    /**
+     * Update the menu items that allow the addition of guide stars. These must be disabled for the automatic group
+     * or if we are currently on the base or a user target.
+     */
+    private void updateGuideStarAdders() {
+        final boolean notBase = !selectionIsBasePosition();
+        final boolean notUser = !selectionIsUserTarget();
+        final boolean notAuto = !(selectionIsAutoGroup() || selectionIsAutoTarget());
+        _guideStarAdders.forEach((gp, comp) -> comp.setEnabled(notBase && notUser && notAuto));
```

Comment:
Might be easier to flip this: `enabled = selectionIsManualGroup() || selectionIsManualTarget()`.